opal/util/stacktrace.c

```diff
@@ -467,23 +467,25 @@ static void show_stackframe(int signo, siginfo_t *info, void *p)
     }
 
     /* write out the signal information generated above */
-    write(opal_stacktrace_output_fileno, print_buffer, sizeof(print_buffer) - size);
+    opal_best_effort_write(opal_stacktrace_output_fileno, print_buffer, sizeof(print_buffer) - size);
```

Comment:
We probably should be checking the return code of this but seeing how write() was not checking it before, this is a good improvement as is. It can be addressed later.

Reply:
I debated about that.  There's nothing we could do with the return code in this case.  We're in a signal handler about to terminate the process.  Our option would be to terminate the process (which we already are going to do).